test/parallel/test-http2-compat-serverrequest-pipe.js

```diff
@@ -10,8 +11,8 @@ const path = require('path');
 
 // piping should work as expected with createWriteStream
 
-const loc = path.join(common.fixturesDir, 'person.jpg');
-const fn = path.join(common.tmpDir, 'http2pipe.jpg');
+const loc = fixtures.path(fixtures.fixturesDir, 'person.jpg');
```

Comment:
Please have a look at `fixtures.path` definition you will see that you don't have to pass the fixturesDir anymore.